DesignSafe-CI/portal
# task/DES-3025 Support OT data in recon portal through Tapipy

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-3025](https://tacc-main.atlassian.net/browse/DES-3025)

## Summary of Changes: ##
Modified the opentopography data urls - `opentopodata_center/` & `opentopo_coordinates/`  to fetch community data using scripts through tapipy

## Testing Steps: ##
1.  Install tapipy using poetry
2.  make build
3.  Resolve dependencies(if any)
4. Confirm all services are running without errors.


## Notes: ##
- [Resolved] Some unit tests are failing with warnings, which are not related to this branch code. Not sure on how to proceed on that.
